cleaners/windows_defender.xml

```diff
@@ -12,20 +12,55 @@
 
     This program is distributed in the hope that it will be useful,
     but WITHOUT ANY WARRANTY; without even the implied warranty of
-    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
     GNU General Public License for more details.
 
     You should have received a copy of the GNU General Public License
-    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+    along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+    @app Microsoft Windows Defender
+    @url https://www.microsoft.com/en-us/safety/pc-security/windows-defender.aspx
+    @os Windows
+    @cleanerversion v1.5.0
+    @cleanerdate 2019-03-13
+    @cleanerby ??? & https://github.com/Tobias-B-Besemer
+    @tested ok v???, Windows 7
+    @testeddate 2019-03-13
+    @testedby https://github.com/Tobias-B-Besemer
+    @note 
 
 -->
 <cleaner id="windows_defender" os="windows">
   <label>Windows Defender</label>
   <description>Anti-virus</description>
   <option id="history">
     <label>History</label>
-    <description>Delete the files</description>
+    <description>Delete the history</description>
```

Comment:
"Delete the history" isn't translated, while "Delete the files" is. The label already clarifies the files are history, so is this a worthwhile change?

Reply:
Sorry, but it's translated!
![2019-03-19 05_12_41-BleachBit](https://user-images.githubusercontent.com/11881073/54579880-d330e680-4a05-11e9-9478-ca33353a2555.jpg)

Reply:
I searched for that German string in `de.po`, and it has the extra word "usage"

```
msgid "Delete the usage history"!
msgstr "Den Verwendungsverlauf löschen"!
```

Reply:
Resolved in 10467a7aba6b8d921ff553bb96a7f9ae56eb7dd9